packages/manager/modules/pci/src/components/pci-order-command-terraform/pci-order-command-terraform.controller.js

```diff
@@ -0,0 +1,43 @@
+export default class PciOrderCommandTerraformCtrl {
+  $onInit() {
+    this.terraformDisplay = this.formatTerraformNode(this.terraformData);
+  }
+
+  formatTerraformNode(node, identationCount = 0) {
+    let keyStr = '';
+    // handle key value
+    if (node.key === 'resource') {
+      keyStr = `${node.key} ${node.values
+        .map((value) => `"${value}"`)
+        .join(' ')}`;
+    } else if (node.values.length > 0) {
+      keyStr = `${node.key} = ${node.values
+        .map((value) => (typeof value === 'string' ? `"${value}"` : value))
+        .join(' ')}`;
+    } else {
+      keyStr = `${node.key}`;
+    }
+    // handle sub nodes
+    if (node.nodes.length > 0) {
+      return `${PciOrderCommandTerraformCtrl.getIdentationString(
+        identationCount,
+      )}${keyStr} {
+${node.nodes
+  .map((subnode) => `${this.formatTerraformNode(subnode, identationCount + 1)}`)
+  .join(`\r\n`)}
+${PciOrderCommandTerraformCtrl.getIdentationString(identationCount)}}`;
+    }
+    return `${PciOrderCommandTerraformCtrl.getIdentationString(
+      identationCount,
+    )}${keyStr}`;
```

Comment:
```suggestion
    return keyStr.padStart(identationCount);
```

Reply:
To be treated with https://jira.ovhcloud.tools/browse/DATATR-236